<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COMP37111 Particle System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
    <item>
     <widget class="OpenGLWidget" name="openGLWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="planetsView"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="eyePosLatLonLabel">
           <property name="text">
            <string>Eye Pos Lat Lon:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="planetsCountLabel">
           <property name="text">
            <string>Planets Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="shadingTestLabel">
           <property name="text">
            <string>Shading Test:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tailLabel">
           <property name="text">
            <string>Tail: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="deathStarLabel">
           <property name="text">
            <string>Death Star:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="renderLabel">
           <property name="text">
            <string>Render:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="axesLabel">
           <property name="text">
            <string>Axes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tpsLabel">
           <property name="text">
            <string>Time Per Frame: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fpsLabel">
           <property name="text">
            <string>FPS: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Gravity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="gravitySlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="speedSlider">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="runButton">
           <property name="text">
            <string>Run</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_Help"/>
    <addaction name="actionCalculate_Ave_FPS"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="actionShow_Help">
   <property name="text">
    <string>Show &amp;Help</string>
   </property>
  </action>
  <action name="actionCalculate_Ave_FPS">
   <property name="text">
    <string>Calculate Ave FPS</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">n_body</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
